packages/aws-cdk/bin/cdk.ts

```diff
@@ -58,7 +58,7 @@ async function parseCommandLineArguments() {
       .option('require-approval', { type: 'string', choices: [RequireApproval.Never, RequireApproval.AnyChange, RequireApproval.Broadening], desc: 'what security-sensitive changes need manual approval' }))
     .command('destroy [STACKS..]', 'Destroy the stack(s) named STACKS', yargs => yargs
       .option('force', { type: 'boolean', alias: 'f', desc: 'Do not ask for confirmation before destroying the stacks' }))
-    .command('diff [STACK]', 'Compares the specified stack with the deployed stack or a local template file', yargs => yargs
+    .command('diff [STACK]', 'Compares the specified stack with the deployed stack or a local template file, and returns with status 1 if any difference is found', yargs => yargs
```

Comment:
Maybe just say nonzero? That gives us some leeway to use exit codes for different cases?

Reply:
I want to specifically mirror the behavior of the POSIX `diff`, which is to return `1` if a difference is found, `0` if no difference is found, and some other value if an error occurred.